spec/features/users/show_spec.rb

```diff
@@ -1,39 +1,45 @@
 # frozen_string_literal: true
 
-require 'rails_helper'
+require "rails_helper"
 
-describe 'Users#show' do
+describe "Users" do
   let(:user_profile) do
     create(:user)
   end
 
-  before(:each) do
+  before do
```

Comment:
I need to fix this rule. You don't have to do it, but I prefer the explicit `before(:each)`.

Reply:
Looks like I can make this work with:

```
RSpec/HookArgument:
  EnforcedStyle: each
```

I'll drop it in.